apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-auto-complete-validation-example.component.html

```diff
@@ -1,35 +1,20 @@
 <div>
-    <fd-popover class="fd-popover" [isOpen]="open">
+    <fd-popover class="fd-popover" #typeahead>
```

Comment:
In the current popover implemetaion in order to have a full control over `open/close` we have to reset default triggers option by assigning `[triggers]="[]"` otherwise each click on `popover-control` will toggle popover anyway.

Reply:
Added `[triggers] = "[]"`.